assistant_dists/dream_multimodal/docker-compose.override.yml

```diff
@@ -1,6 +1,6 @@
 services:
   agent:
-    command: sh -c 'bin/wait && python -m deeppavlov_agent.run agent.pipeline_config=assistant_dists/dream_multimodal/pipeline_conf.json'
+    command: sh -c 'bin/wait && python -m deeppavlov_agent.run agent.channel=telegram agent.telegram_token=$TG_TOKEN agent.pipeline_config=assistant_dists/dream_multimodal/pipeline_conf.json'
```

Comment:
no, we have a separate file for command for telegram actually:
telegram.yml

Reply:
in every dist

Reply:
Done